DependencyInjection/Configuration.php

```diff
@@ -388,6 +388,7 @@ private function addExceptionSection(ArrayNodeDefinition $rootNode)
                             ->useAttributeAsKey('name')
                             ->prototype('boolean')->end()
                         ->end()
+                        ->booleanNode('debug')->treatNullLike(false)->end()
```

Comment:
I would also inject the `kernel.debug` parameter to the `Configuration` constructor (for example, in Symfony we do the same in the FrameworkBundle: https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L197).